src/error.rs

```diff
@@ -154,6 +158,9 @@ pub(crate) enum ErrorCode {
 
     /// Unknown event has been requested.
     UnknownEvent(String),
+
+    /// Issued command failed.
+    CommandFailed(String)
```

Comment:
When `errmsg` is missing, it should be `None` rather than `"unknown"` generated by this library.

```suggestion
    CommandFailed(Option<String>),
```

Reply:
Done